### PR TITLE
Milestone #2 User Interface

### PR DESCRIPTION
**Pull request by denisdiaconu**
The user interface of the TicTacToe project.
We build the user interface in the main.rb file.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
**Invalid** :red_circle: 
Hello,
Your Pull Requests should have a meaningful title and short summary:
### REQUIRED
You should make sure that your repo looks professional before review.
Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Happy coding
Rindra Josia

**Review by mke2111 — CHANGES_REQUESTED** [review]
Kindly fix all the linter errors you are currently having

**Comment by denisdiaconu** [comment]
Thank you for the rev!
fixed all the linters errors

**Review by mke2111 — CHANGES_REQUESTED** [review]
# **🔴 STATUS: CHANGES REQUIRED 🔴 ⚠️** 

## Hey @denisdiaconu  & @nganifaith , 👋🏽 
 **Thank you for the good job you got going on. The game is starting to take shape and the mockup flow is almost perfect. But we have a few changes to make to get everything right.**

## REQUIRED CHANGES

- [ ] This milestone should be a continuation from milestone 1. The main file should still be executable.

- [ ] The game informs the player if the selected move is invalid (without computing what an invalid move is). Currently all user inputs are accepted. Should accept only integers, otherwise should inform the player about the invalid move.
- [ ] The game informs the player if the selected move is a winning move (without computing the winning move).
- [ ] The game informs the player if the selected move is a draw move (without computing draw).
- [ ] The game repeats all actions for the next player's move. This should go on until there's a win or a draw.
- [ ] A loop is needed in this game (recommended is while but anything else will do). This will help implement some of the above requirements.
```
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end

```

**Kindly make all the requested changes and submit another CRR. 👍🏽** 

## Cheers!! 🍹 & Happy Coding. 💻 

# Reach me on slack for a chitchat 🆘  **Roy Mukuye**

File: bin/main.rb [hunk 1]
```diff
@@ -1,4 +1,37 @@
 require_relative '../lib/game_logic'
```
Comment (mke2111) [inline bin/main.rb:3]: The first line should ensure that the file is executable

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUESTED 🔴 ♻️ 

Hey team @denisdiaconu and @nganifaith 👋🏾 , good job so far with the project. However, the main file is still not executable, so before adding more comments kindly work on **ALL** changes the previous TSE requested before submitting another CRR.


 That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help_
Regards
Peris Ndanu

**Review by GzimAsani — APPROVED** [review]
# Status: APROVED  :tada: :tada: 
**Hello team** very good job, you did a great work implementing the changes from the previous reviewers:

### However some of your code was beyond the scope of this millstone so I did not review it. Keep in mind for the next times to not start another millstone before finishing the first one.

### HAPPY CODING :computer:

**Pull request merged**